public-api/db-cdr/generate-cdr/run-achilles-queries.sh

```diff
@@ -306,7 +306,7 @@ CAST(floor((extract(year from condition_start_date) - p1.year_of_birth)/10) AS S
 COUNT(distinct p1.person_id) as count_value,
 COUNT(distinct p1.person_id) as source_count_value from \`${BQ_PROJECT}.${BQ_DATASET}.person\` p1 inner join \`${BQ_PROJECT}.${BQ_DATASET}.condition_occurrence\` co1
 on p1.person_id = co1.person_id where floor((extract(year from condition_start_date) - p1.year_of_birth)/10) >=3 and floor((extract(year from condition_start_date) - p1.year_of_birth)/10) < 9
-and co1.condition_concept_id != co1.condition_source_concept_id
+and co1.condition_source_concept_id not in (select distinct condition_concept_id from \`${BQ_PROJECT}.${BQ_DATASET}.condition_occurrence\`)
```

Comment:
The concept can be source concept with different condition_concept_id but this creates duplicate rows for same bin. Checking if it is not the condition_concept_id anywhere to avoid duplicates. 
Tested this query works fine.